modules/setting/lfs.go

```diff
@@ -37,40 +37,15 @@ func newLFSService() {
 	}
 
 	lfsSec := Cfg.Section("lfs")
-	LFS.Storage.Type = lfsSec.Key("STORAGE_TYPE").MustString("")
-	if LFS.Storage.Type == "" {
-		LFS.Storage.Type = "default"
-	}
-
-	if LFS.Storage.Type != LocalStorageType && LFS.Storage.Type != MinioStorageType {
-		storage, ok := storages[LFS.Storage.Type]
-		if !ok {
-			log.Fatal("Failed to get lfs storage type: %s", LFS.Storage.Type)
-		}
-		LFS.Storage = storage
-	}
+	storageType := lfsSec.Key("STORAGE_TYPE").MustString("")
 
-	// Override
-	LFS.ServeDirect = lfsSec.Key("SERVE_DIRECT").MustBool(LFS.ServeDirect)
-	switch LFS.Storage.Type {
-	case LocalStorageType:
-		// keep compatible
-		LFS.Path = sec.Key("LFS_CONTENT_PATH").MustString(filepath.Join(AppDataPath, "lfs"))
-		LFS.Path = lfsSec.Key("PATH").MustString(LFS.Path)
-		if !filepath.IsAbs(LFS.Path) {
-			LFS.Path = filepath.Join(AppWorkPath, LFS.Path)
-		}
+	// Specifically default PATH to LFS_CONTENT_PATH
+	lfsSec.Key("PATH").MustString(
```

Comment:
This is not equal `sec.Key("LFS_CONTENT_PATH").MustString(filepath.Join(AppDataPath, "lfs"))`

Reply:
Agreed it is not.

But it is the same if LFS_CONTENT_PATH is set.

And it is the same if PATH is set.

The only difference is if you have:

```ini
[lfs]
STORAGE_TYPE=my-special-lfs-storage
; I have not set LFS_CONTENT_PATH or PATH here because I want to set that in my-special-lfs-storage

[storage.my-special-lfs-storage]
PATH= put-my-lfs-files-here
```

Under the previous code the `PATH` in `[storage.my-special-lfs-storage]` would be always overridden by the default `LFS_CONTENT_PATH`

Reply:
`LFS_CONTENT_PATH` is in `server` section, now we have `lfs` section and `[storage.my-special-lfs-storage] `.